census/src/main/scala/hmda/census/model/TractLookup.scala

```diff
@@ -1,17 +1,20 @@
 package hmda.census.model
 
+import com.github.tototoshi.csv.CSVParser._
+
 // The file contains a list of all census tracts and what county and state they are in
 // site: https://www.census.gov/geo/maps-data/data/relationship.html
 // file: http://www2.census.gov/geo/docs/maps-data/data/rel/trf_txt/us2010trf.txt
 
 object TractLookup extends CbsaResourceUtils {
   val values: Set[Tract] = {
-    val lines = csvLines("/us2010trf.txt")
```

Comment:
Do we still need `us2010trf.txt` in the `/resources` folder, or can we delete it?